src/app/shared/support/support.html

```diff
@@ -0,0 +1,48 @@
+<div class="docs-help-support">
+  <ul>
+    <li>
+      <a href="https://stackoverflow.com/questions/tagged/angular-material">
+        <img src="../../../assets/img/homepage/stackoverflow-logo.svg" alt="StackOverflow icon">
```

Comment:
These `img` tags should set the `loading="lazy"` attribute

Reply:
Done!